### ISSUE TITLE
Accessibility for the pagination snippet

### ISSUE THREAD

**Issue by littlesvensson** [DefectDojo/django-DefectDojo#11590]
**Bug description**

Screen reader users cannot perceive the pagination correctly because the individual numbers are not labelled in a meaningful way. These are displayed/read as ‘1 link’, for example. As a result, their function is not recognisable for blind users. The same is valid for dropdown menu button where user can choose how many items shall be displayed per page.

**Steps to reproduce**

1. Go to any page where pagination is present.
2. Let screenreader read what it "sees". For the example in the screenshot, its "Showing entries 1 to 25 of 79 1 2 3 4 next page size toggle dropdown" (using Screen Reader Chrome extension and MacOS VoiceOver Utility)

![Image](https://github.com/user-attachments/assets/2c91071b-a070-473f-baed-be9dfdf4c33b)

**Expected behavior**
Screen reader should provide information understandable to its targer audience - meaningful information like "pagination navigation" "page 4", 25 items per page" etc.

**Comment by manuel-sommer** [comment]
can we close this @valentijnscholten ?
